src/js/facility-locator/reducers/facilities.js

```diff
@@ -1,5 +1,3 @@
-import { FETCH_VA_FACILITY, FETCH_VA_FACILITIES, SEARCH_FAILED } from '../actions';
```

Comment:
There are some strange errors from Babel happening, when I include these so I removed their usage. I spent a bunch of time debugging this to no avail; we can remove these to your point about scaling. I'll remove the usage in the `reducers/index`